include/cantera/thermo/MixtureFugacityTP.h

```diff
@@ -1,577 +1,572 @@
+    //! given temperature
+    /*!
+     * @param TKelvin   Temperature (Kelvin)
+     * @return          The saturation pressure at the given temperature
+     */
+    virtual doublereal satPressure(doublereal TKelvin);
+	virtual void setToEquilState(const doublereal* lambda_RT);
+    virtual void getActivityConcentrations(double* c) const;
+
+protected:
+    //! Calculate the pressure given the temperature and the molar volume
+    /*!
+     * @param   TKelvin   temperature in kelvin
+     * @param   molarVol  molar volume ( m3/kmol)
+     * @returns the pressure.
+     */
+    virtual doublereal pressureCalc(doublereal TKelvin, doublereal molarVol) const;
+
+    //! Calculate the pressure and the pressure derivative given the temperature
+    //! and the molar volume
+    /*!
+     *  Temperature and mole number are held constant
+     *
+     * @param   TKelvin   temperature in kelvin
+     * @param   molarVol  molar volume ( m3/kmol)
+     * @param   presCalc  Returns the pressure.
+     * @returns the derivative of the pressure wrt the molar volume
+     */
+    virtual doublereal dpdVCalc(doublereal TKelvin, doublereal molarVol, doublereal& presCalc) const;
+
+    virtual void updateMixingExpressions();
+
+
+    //! Solve the cubic equation of state
+    /*!
+     * 
+     * Returns the number of solutions found. If it only finds the liquid
+     * branch solution, it will return a -1 or a -2 instead of 1 or 2.  If it
+     * returns 0, then there is an error.
+     * The cubic equation is solved using Nickall's method (Ref: The Mathematical Gazette(1993), 77(November), 354�359, https://www.jstor.org/stable/3619777)
+     */
+    int solveCubic(double T, double pres, double a, double b, 
+                                        double aAlpha, double Vroot[3], double an,
+                                        double bn, double cn, double dn, double tc, double vc) const;
+
+    //@}
+
+protected:
+    //! Storage for the current values of the mole fractions of the species
+    /*!
+     * This vector is kept up-to-date when some the setState functions are called.
+     */
+    vector_fp moleFractions_;
+
+    //! Current state of the fluid
+    /*!
+     *  There are three possible states of the fluid:
+     *  - FLUID_GAS
+     *  - FLUID_LIQUID
+     *  - FLUID_SUPERCRIT
+     */
+    int iState_;
+
+    //! Force the system to be on a particular side of the spinodal curve
+    int forcedState_;
+
+    //! Temporary storage for dimensionless reference state enthalpies
+    mutable vector_fp m_h0_RT;
+
+    //! Temporary storage for dimensionless reference state heat capacities
+    mutable vector_fp m_cp0_R;
+
+    //! Temporary storage for dimensionless reference state Gibbs energies
+    mutable vector_fp m_g0_RT;
+
+    //! Temporary storage for dimensionless reference state entropies
+    mutable vector_fp m_s0_R;
+
+public:
+	//! Temporary storage - length = m_kk.

```

Comment:
You have some tabs here instead of spaces. You can configure your editor to put spaces when you push the tab key, please do that to save yourself some trouble. If you need advice on how to do that, let me know!

Reply:
Corrected.